server/lib/update-fields.js

```diff
@@ -0,0 +1,40 @@
+import Customer from '../models/customer'
+import Volunteer from '../models/volunteer'
+
+export default {
+  /**
+   * Update Volunteer fields of address
+   */
+  async updatingVolunteer(req) {
+    //Updating address of the volunteer role if exists
+    const volunteer = await Volunteer.findById(req.body._id)
+    if(volunteer != undefined && volunteer != null && volunteer != {}){
+      let oldAdrFields = []
```

Comment:
In the case of the name I do prefer a general name that tomorrow make more simple the re-utilization of the function for the donor controller, is that ok for you? But I am deleting the "let oldAdrFields = []" does not have sense there.